micrometer-spring-legacy/src/test/java/io/micrometer/spring/web/servlet/WebMvcMetricsIntegrationTest.java

```diff
@@ -81,7 +81,7 @@ public void handledExceptionIsRecordedInMetricTag() throws Exception {
 
         clock.add(SimpleConfig.DEFAULT_STEP);
         assertThat(this.registry.find("http.server.requests")
-            .tags("exception", "Exception1").value(Statistic.Count, 1.0).timer())
+            .tags("exception", "Exception1", "status", "500").value(Statistic.Count, 1.0).timer())
```

Comment:
This succeeds for me. So no problem here.

Reply:
Correct. I just added it because I believe it needs to be asserted that the right tags are being applied.

Reply:
Look to the test name here. The point of the test is to prove that we record a handled exception that still returns a 200. This change is invalid.